[greenpepper-license] dependency not found

The libraries **truelicense:truexml** and **truelicense:truelicense** could not be resolved.

We can use:

```xml
<dependency>
	<groupId>de.schlichtherle.truelicense</groupId>
	<artifactId>truelicense-core</artifactId>
	<version>1.33</version>
</dependency>

<dependency>
	<groupId>de.schlichtherle.truelicense</groupId>
	<artifactId>truelicense-xml</artifactId>
	<version>1.33</version>
</dependency>
```

```log
[ERROR] Failed to execute goal on project greenpepper-license: Could not resolve dependencies for project greenpepper:greenpepper-license:jar:4.0-SNAPSHOT: The following artifacts could not be resolved: truelicense:truelicense:jar:1.29, truelicense:truexml:jar:1.29: Could not find artifact truelicense:truelicense:jar:1.29 in atlassian-public-m1 (https://maven.atlassian.com/content/groups/public) -> [Help 1]
```

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,0 +1,2 @@
+FROM maven:3.2-jdk-7-onbuild
+

--- a/README.md
+++ b/README.md
@@ -9,3 +9,11 @@
 ### Comment and Communicate on [![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/strator-dev/greenpepper?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
 ### Submit patches/merge request by forking the code
+
+# Build and get the jar
+
+```sh
+docker build --tag green-build .
+docker run -d --name green green-build true
+docker cp green:/usr/src/app/greenpaper-confluence/greenpaper-confluence5-plugin/target/greenpepper-confluence5-plugin-4.0-SNAPSHOT.jar .
+```

--- a/greenpaper-confluence/greenpaper-confluence3-plugin/src/main/assembly/dep.xml
+++ b/greenpaper-confluence/greenpaper-confluence3-plugin/src/main/assembly/dep.xml
@@ -22,8 +22,8 @@
 				<include>org.hibernate:hibernate-commons-annotations</include>
 				<include>org.hibernate:ejb3-persistence</include>
 				<!--  <include>javax.persistence:ejb</include>  -->
-				<include>truelicense:truelicense</include>
-				<include>truelicense:truexml</include>
+				<include>de.schlichtherle.truelicense:truelicense-core</include>
+				<include>de.schlichtherle.truelicense:truelicense-xml</include>
 			</includes>
 			<unpack>true</unpack>
 			<unpackOptions>

--- a/greenpaper-confluence/greenpaper-confluence4-plugin/src/main/assembly/dep.xml
+++ b/greenpaper-confluence/greenpaper-confluence4-plugin/src/main/assembly/dep.xml
@@ -22,8 +22,9 @@
 				<include>org.hibernate:hibernate-commons-annotations</include>
 				<include>org.hibernate:ejb3-persistence</include>
 				<!--  <include>javax.persistence:ejb</include>  -->
-				<include>truelicense:truelicense</include>
-				<include>truelicense:truexml</include>
+<include>de.schlichtherle.truelicense:truelicense-core</include>
+<include>de.schlichtherle.truelicense:truelicense-xml</include>
+
 			</includes>
 			<unpack>true</unpack>
 			<unpackOptions>

--- a/greenpaper-confluence/greenpaper-confluence5-plugin/src/main/assembly/dep.xml
+++ b/greenpaper-confluence/greenpaper-confluence5-plugin/src/main/assembly/dep.xml
@@ -22,9 +22,9 @@
 				<include>org.hibernate:hibernate-commons-annotations</include>
 				<include>org.hibernate:ejb3-persistence</include>
 				<!--  <include>javax.persistence:ejb</include>  -->
-				<include>truelicense:truelicense</include>
-				<include>truelicense:truexml</include>
-			</includes>
+			<include>de.schlichtherle.truelicense:truelicense-core</include>
+<include>de.schlichtherle.truelicense:truelicense-xml</include>
+                        </includes>
 			<excludes>
 				<exclude>greenpepper:greenpepper-client:jar:complete</exclude>
 				<exclude>greenpepper:greenpepper-confluence-demo:jar:fixtures</exclude>

--- a/greenpaper/greenpepper-license/pom.xml
+++ b/greenpaper/greenpepper-license/pom.xml
@@ -27,17 +27,20 @@
             <artifactId>commons-io</artifactId>
             <version>1.2</version>
         </dependency>
-        <dependency>
-            <groupId>truelicense</groupId>
-            <artifactId>truelicense</artifactId>
-            <version>1.29</version>
-        </dependency>
-        <dependency>
-            <groupId>truelicense</groupId>
-            <artifactId>truexml</artifactId>
-            <version>1.29</version>
-        </dependency>
-        <dependency>
+        
+<dependency>
+        <groupId>de.schlichtherle.truelicense</groupId>
+        <artifactId>truelicense-core</artifactId>
+        <version>1.33</version>
+</dependency>
+
+<dependency>
+        <groupId>de.schlichtherle.truelicense</groupId>
+        <artifactId>truelicense-xml</artifactId>
+        <version>1.33</version>
+</dependency>
+
+<dependency>
             <groupId>greenpepper</groupId>
             <artifactId>greenpepper-client</artifactId>
             <version>${project.version}</version>

